consensus/src/consensus.rs

```diff
@@ -342,6 +344,13 @@ impl<R: ConsensusRpc> ConsensusClient<R> {
         self.store.current_sync_committee = bootstrap.current_sync_committee;
         self.store.optimistic_header = bootstrap.header.clone();
 
+        // force to initialize mmr storage with on-chain base slot
+        if !self.storage().is_initialized()? {
+            let header = self.rpc.get_header(base_slot).await?;
+            let update = Update::from_finalized_header(header);
+            self.store_finalized_update(&update, true)?;
+        }
+
```

Comment:
Is this block duplicated if [MMR was updated here](https://github.com/synapseweb3/forcerelay-verifier/pull/9/files#r1102248465)?

Reply:
no, because I force the slots in storage must be strictly continuous through `store_updates_until_finality_update()`, by the way, our test case passes to proving this.